### PR TITLE
Update model_runner_cpp.py

### PR DESCRIPTION
**Pull request by RoyHe**
Gather generation logits and gather context logits can be setup separately

### CODE REVIEW

**Comment by byshiue** [comment]
Thank you for the PR. The issue is fixed in latest main branch. Close it.
